dist/getting-started/getting-started.php

```diff
@@ -47,8 +47,8 @@ function atomic_blocks_getting_started_menu() {
 	$icon_url   = 'dashicons-screenoptions';
 
 	if ( atomic_blocks_is_pro() ) {
-		$page_title = esc_html__( 'Genesis Page Builder Settings', 'atomic-blocks' );
-		$menu_title = esc_html__( 'Page Builder', 'atomic-blocks' );
+		$page_title = esc_html__( 'Genesis Blocks Pro Settings', 'atomic-blocks' );
+		$menu_title = esc_html__( 'Genesis Blocks', 'atomic-blocks' );
```

Comment:
@mindctrl Looks like this innocent menut title change causes the “Block Permission Settings” tab to fail to load.

It's because we have these checks in Genesis Page Builder:

- https://github.com/studiopress/genesis-page-builder/blob/c23f4f336b22b4bae63a6d52ed199cb4e6c57e54/genesis-page-builder.php#L171-L173
- https://github.com/studiopress/genesis-page-builder/blob/c23f4f336b22b4bae63a6d52ed199cb4e6c57e54/genesis-page-builder.php#L221-L223

And the title change here alters the current screen id:

```diff
-page-builder_page_atomic-blocks-plugin-settings
+genesis-blocks_page_atomic-blocks-plugin-settings
```

We could update https://github.com/studiopress/genesis-page-builder/pull/98 to look for the new IDs?

Reply:
Oof good catch! I'll update that.

Reply:
@nickcernis updated in https://github.com/studiopress/genesis-page-builder/pull/98/commits/588cb96350fba7b8b5d389129cd1587e0b235c06